cvr/pom.xml

```diff
@@ -67,7 +67,7 @@
     <dependency>
       <groupId>com.github.genome-nexus</groupId>
       <artifactId>genome-nexus-annotation-pipeline</artifactId>
-      <version>48474a5d86</version>
+      <version>164ff61f07761423c129571e5364085b01e9a84d</version>
```

Comment:
this will just need to get updated once the GN annotation pipeline PR gets merged - I will give @sheridancbio a chance to review first though